### PR TITLE
Friend page

### PR DESCRIPTION
**Pull request by Lunilux24**
adding friend page to main branch

### CODE REVIEW

**Comment by Lunilux24** [comment]
The main issue completed this week was the addition of the friend page. This will be where users can direct message each other as well as use call and video. this also leads to the adding friends page where users can add each other.

Issues completed this week:
- Friend Page HTML
- Friend Page CSS
- Add Friend Page HTML
- Add Friend Page CSS
- Add Friend Page Client-Side Form Validation

Issues requiring completion before final submission:
- Link all pages properly
- Ensure consistent styling across all pages
- HTML/CSS testing for the 2 newly created pages

File: DiscordCode/FrontEnd/views/addFriend.ejs [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
```
Comment (jordanp2002) [inline DiscordCode/FrontEnd/views/addFriend.ejs:1]: Looks good!

File: DiscordCode/FrontEnd/css/addFriend.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
+body {
```
Comment (jordanp2002) [inline DiscordCode/FrontEnd/css/addFriend.css:1]: Looks good, may have to make adjusts to some colors later to provide overall consistency to the system.

**Review by jordanp2002 — APPROVED** [review]

File: DiscordCode/FrontEnd/css/friend_page.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
+.friend_list {
```
Comment (jordanp2002) [inline DiscordCode/FrontEnd/css/friend_page.css:1]: Looks good, may have to change some colors to provide overall consistency across the system.

File: DiscordCode/FrontEnd/views/friend_page.html [hunk 4]
```diff
@@ -9,43 +9,54 @@
 </head>
```
Comment (jordanp2002) [inline DiscordCode/FrontEnd/views/friend_page.html]: looks good!

**Review by jordanp2002 — APPROVED** [review]
Overall code looks well formatted and css is consistent across the given pages. Minor adjusmtments to css elements like colors and shadows may be needed for final submission to provide consistency across the system.

**Pull request merged**